spec/factories.rb

```diff
@@ -1,4 +1,14 @@
 FactoryBot.define do
+  factory :campaign do
+    title 'MyString'
+    description 'MyText'
+    status 'MyString'
+    current_amount 1
+    needed_amount 1
+    requisite 'MyString'
+    category_id 1
+    finished_at Time.now
```

Comment:
it's better to use it with `zone` because tests may fail if someone run them from another time zone.